packages/metal-dom/src/dom.js

```diff
@@ -254,6 +254,31 @@ class dom {
 	}
 
 	/**
+	 * Verifies if the element is able to trigger the Click event,
+	 * simualting browsers behaviour, avoiding event listeners to be called by dom.triggerEvent method.
+	 * @param {Element} node Element to be checked.
+	 * @param {!String} eventName The event name.
+	 */
+	static isAbleToInteract_(node, eventName) {
+		var currElement = node;
```

Comment:
Both `currElement` and `matchesSelector` are only used inside the if clause, so it'd be best to move them there (we don't require var declarations at the top of the function for Metal.js, we instead prefer them to be near their first usage. If you use `let` or `const` instead of  `var` this is even better.